paimon-flink/paimon-flink-cdc/src/test/java/org/apache/paimon/flink/action/cdc/kafka/KafkaActionITCaseBase.java

```diff
@@ -221,8 +220,6 @@ public KafkaSyncDatabaseActionBuilder(Map<String, String> kafkaConfig) {
     }
 
     protected void createTestTopic(String topic, int numPartitions, int replicationFactor) {
-        Map<String, Object> properties = new HashMap<>();
-        properties.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, getBootstrapServers());
```

Comment:
The `getBootstrapServers()` method could be removed as well.

Reply:
Thanks for your review! done.